jest.config.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
-	setupFilesAfterEnv: ['./jest.setup.js', './puppeteer.setup.js']
+	setupFilesAfterEnv: ['./jest.setup.js', './puppeteer.setup.js'],
+	testEnvironment: 'jsdom'
```

Comment:
This was added because of jest version 27
https://stackoverflow.com/questions/46274889/jest-test-fails-with-window-is-not-defined